functions/src/export-csv.ts

```diff
@@ -104,8 +104,8 @@ export async function exportCsvHandler(
       row.push(location['_latitude'] || '');
       row.push(location['_longitude'] || '');
       row.push(toWkt(loi.get('geoJson')) || '');
-      const responses = submission['responses'] || {};
-      tasks.forEach((task, taskId) => row.push(getValue(taskId, task, responses)));
+      const data = submission['data'] || {};
```

Comment:
Should we also maintain backwards compatibility with 'responses' here?

Please add TODO with reference to #1288 to clean this up.